<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HW_8_QTClass</class>
 <widget class="QMainWindow" name="HW_8_QTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HW_8_QT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>691</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Meno:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>20</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Skóre:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Úroveň:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="newGameButton">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>20</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Nová hra</string>
         </property>
        </widget>
        <widget class="QPushButton" name="endGameButton">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>20</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Ukončit hru</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="randomOrderCheckBox">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>20</y>
           <width>77</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Náhodné</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>113</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="difficultyComboBox">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>20</y>
           <width>72</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="scoreSpinBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>20</y>
           <width>62</width>
           <height>24</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>691</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Žolíky</string>
        </property>
        <widget class="QPushButton" name="jokerButton1">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>30</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>50:50</string>
         </property>
        </widget>
        <widget class="QPushButton" name="jokerButton2">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>50:50</string>
         </property>
        </widget>
        <widget class="QPushButton" name="jokerButton3">
         <property name="geometry">
          <rect>
           <x>440</x>
           <y>30</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>50:50</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>691</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>Otázka</string>
        </property>
        <widget class="QPlainTextEdit" name="questionTextEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>671</width>
           <height>121</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>310</y>
          <width>691</width>
          <height>80</height>
         </rect>
        </property>
        <property name="title">
         <string>Odpovede</string>
        </property>
        <widget class="QRadioButton" name="answerARadioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>A)</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="answerBRadioButton">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>40</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>B)</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="answerCRadioButton">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>40</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>C)</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="answerDRadioButton">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>40</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>D)</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>691</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="confirmButton">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>20</y>
           <width>80</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Potvrdiť</string>
         </property>
        </widget>
        <widget class="QPushButton" name="skipButton">
         <property name="geometry">
          <rect>
           <x>589</x>
           <y>20</y>
           <width>101</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Preskočit otázku</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HW_8_QT.qrc"/>
 </resources>
 <connections/>
</ui>
